app/addons/auth/components.react.jsx

```diff
@@ -239,9 +238,6 @@ define([
             <input id="password" type="password" name="password" placeholder= "Password" size="24"
               onChange={this.onChangePassword} />
             <p>
-              Non-admin users have read and write access to all databases, which
-              are controlled by validation functions. CouchDB can be configured to block all
-              access to anonymous users.
```

Comment:
Hello. :) I didn't mean to delete the paragraph. I meant the new wording needs some improvements.

What was wrong with this original wording? I like it.

As Kxepal mentioned, it might be helpful to point the user to where/how they can protect their database(s). Perhaps a helpful link in a circle-question-mark-symbol, to the correct place in the docs?